OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -19,7 +19,7 @@
 
 namespace OpenRA.Traits
 {
-	// depends on the order of pips in WorldRenderer.cs!
+	// depends on the order of pips in SelectionDecorations.cs!
```

Comment:
As you're already editing this, start the comment with an uppercase letter, please.